kayvee.go

```diff
@@ -19,7 +20,11 @@ const (
 
 // Format converts a map to a string of space-delimited key=val pairs
 func Format(data map[string]interface{}) string {
-	formattedString, _ := json.Marshal(data)
+	formattedString, err := json.Marshal(data)
+	if err != nil {
+		log.Printf("Error converting kayvee message to json: %s", err.Error())
```

Comment:
If you aren't calling it with the input it should be fine, right? e.g. you know that `err.Error()` is a string.